requirements-selftests.txt

```diff
@@ -34,3 +34,6 @@ lxml==4.2.4
 # in make develop. The proper solution would be migrate
 # our make develop process to use pip
 libvirt-python==4.6.0
+
+# For avocado.utils.network selftests
+netifaces
```

Comment:
We need to either make the test optional on the presence or not of the `netifaces` module, or add it as a hard build requirement also on the RPM spec file.